schemas/view-schema.json

```diff
@@ -199,16 +194,44 @@
                 }
               },
               "readOnly": true
+            },
+            "focused_times": {
+              "type": "array",
+              "description": "List of the time the user had the document focused for browser",
+              "items": {
+                "type": "object",
+                "description": "Properties of the focused times of the view",
+                "required": ["duration", "start"],
+                "properties": {
+                  "start": {
+                    "type": "integer",
+                    "description": "Timestamp when the user gain focus of the document",
+                    "minimum": 0,
+                    "readOnly": true
+                  },
+                  "duration": {
+                    "type": "integer",
+                    "description": "duration in ns when the user gain focus of the document",
+                    "minimum": 0,
+                    "readOnly": true
+                  },
+                  "currently_focused": {
```

Comment:
is it different than `view.focus.has_focused`?

Reply:
`view.focus.has_focus` would tell if the view has focus when it opened.
`currently_focused` will give you the change in focus during the life of the view.

Reply:
got it, could it be more accurate to rename `has_focus` in `start_focused` then?